StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1507CodeMustNotContainMultipleBlankLinesInARow.cs

```diff
@@ -54,16 +57,54 @@ public class SA1507CodeMustNotContainMultipleBlankLinesInARow : DiagnosticAnalyz
         /// <inheritdoc/>
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
         {
-            get
-            {
-                return SupportedDiagnosticsValue;
-            }
+            get { return SupportedDiagnosticsValue; }
```

Comment:
I don't think so and it should not either

See excerpt from SA1502 documentation below:
> As an exception to this rule, accessors within properties, events, or indexers are allowed to be written all on a single line, as long as the accessor is short.